### DISCUSSION TITLE
Picocom + ESP8266 + Arduino IDE Issue

### DISCUSSION THREAD

**Discussion by tsimur** [micropython/micropython#14304] [ESP8266]
Today I tried micropython for the first time. I followed the steps described in the documentation to flash the board with the latest firmware. I suppose I succeeded, but something bothers me. The documentation suggests using the `picocom /dev/ttyUSB0 -b115200` command to get the REPL. When I do this, picocom does not show anything until I launch the Arduino IDE and open the serial port in it. After this, `>>` appears in picocom and I can successfully use the interpreter. If any of you have encountered this and found a solution, I'd greatly appreciate your insights.

Used commands:
```
esptool.py --port /dev/ttyUSB0 erase_flash 567 esptool.py --port /dev/ttyUSB0 --baud 115200 write_flash --flash_size=detect 0 ~/Downloads/ESP8266_GENERIC-20240222-v1.22.2.bin

# Alternative try for flashing
esptool.py --port /dev/ttyUSB0 --baud 460800 write_flash --flash_size=detect -fm dout 0 ~/Downloads/ESP8266_GENERIC-20240222-v1.22.2.bin

picocom -b 115200 -r -l /dev/ttyUSB0
```

OS: Ubuntu2204
BOARD: No-name board from Aliexpress with ESP8266MOD
FLASH: 4MB

**Comment by shariltumin** [comment]
Have you try pressing the **Enter** key a few times? This will give you the prompt >>>.

You can also try:

```bash
$ tio /dev/ttyUSB0
[tio 21:50:26] tio v1.32
[tio 21:50:26] Press ctrl-t q to quit
[tio 21:50:26] Connected

>>> 
>>> 
>>> 
```

or 
```bash
$ miniterm  /dev/ttyUSB0 115200
--- Miniterm on /dev/ttyUSB0  115200,8,N,1 ---
--- Quit: Ctrl+] | Menu: Ctrl+T | Help: Ctrl+T followed by Ctrl+H ---

>>> 
>>> 
```

or better
```bash
$ mpremote u0
Connected to MicroPython at /dev/ttyUSB0
Use Ctrl-] or Ctrl-x to exit this shell

>>> 
>>> 
```

**Reply by tsimur** [reply]
Yes I have tried pressing the Enter key a few times.
I tried `tio and mpremote` which you suggested. The situation remains the same, but there is one difference. These tools do not continue to work after I close the Arduino serial port. Perhaps information about this difference will be useful.

**Reply by tsimur** [reply]
I also have tried using a different instance of the board and a different PC to eliminate the influence of the OS environment and accidental failure of the board. The result is the same.

**Reply by shariltumin** [reply]
What are your account settings? Do you belong to the **dialout** group?

**Comment by jdkratz** [comment]
I ran into a similar situation, and managed to resolve it by lowering the RTS and DTR settings as below when starting Picocom:


picocom --lower-rts --lower-dtr -b 115200 /dev/ttyUSB0
